src/pvrclient-argustv.cpp

```diff
@@ -867,6 +868,62 @@ PVR_ERROR cPVRClientArgusTV::SetRecordingPlayCount(const kodi::addon::PVRRecordi
   return PVR_ERROR_NO_ERROR;
 }
 
+// GetRecordingEdl source borrowed from pvr.wmc
+PVR_ERROR cPVRClientArgusTV::GetRecordingEdl(const kodi::addon::PVRRecording& recording,
+                                             std::vector<kodi::addon::PVREDLEntry>& edl)
+{
+  if (!FindRecEntry(recording.GetRecordingId(), _streamFileName))
```

Comment:
streamFilename is only used in this function so why not declare it here instead of in the header?

`std::string streamFilename; `